packages/lifecycle/src/filters/attributesFilter.ts

```diff
@@ -0,0 +1,41 @@
+/* ========= FILTERS ========= */
+
+import {
+  CertifiedTenantAttribute,
+  CompactTenant,
+  DeclaredTenantAttribute,
+  Tenant,
+  TenantId,
+  VerifiedTenantAttribute,
+  tenantAttributeType,
+} from "pagopa-interop-models";
+
+export const filterVerifiedAttributes = (
+  producerId: TenantId,
+  tenant: Tenant | CompactTenant
```

Comment:
should be enough (same for others)
where `CompactTenant` will be the new model in lifecycle with the new name
```suggestion
  tenant: CompactTenant
```

Reply:
as replied [here](https://github.com/pagopa/interop-be-monorepo/pull/698#discussion_r1665909481) it will be changed with a new type.